### ISSUE TITLE
File request following issue #7198

### ISSUE THREAD

**Issue by jnguyenv** [obophenotype/human-phenotype-ontology#7339]
Hello HPO Team,

I am the intern working at Inserm under the supervision of @maudDLVT.
@matentzn  You provided @maudDLVT an Excel file with all the HPO codes under HP:0030057 a few months ago and I would like to get the same file. I am asking you again to get the most up-to-date data in case there's been any update since @maudDLVT asked.
Would it be possible to get the following columns in the file :
- hpo code
- label
- synonym
- description
- orpha code
- pmid

Thank you very much in advance for your help.
Julien

**Comment by matentzn** [comment]
I re-ran the report but nothing has changed compared to https://github.com/obophenotype/human-phenotype-ontology/issues/7198#issuecomment-961881326

**Comment by matentzn** [comment]
Ah I see, you want different properties.. Let me see what I can do.

**Comment by matentzn** [comment]
Can you check this and see wether this works for you:

https://api.triplydb.com/s/vuJ28yNBV

I can create the excel file if the result looks ok.

There is no Orpha codes in HPO!

**Comment by julienN75** [comment]
The query does not work for me. It displays this error : "Bad Gateway (#502)".

**Comment by julienN75** [comment]
Could you please tell me more about how I can get orpha codes thanks to HPOA ?

**Comment by matentzn** [comment]
Sorry. The database endpoint must have been switched off: Try https://api.triplydb.com/s/XplI5LTcn

**Comment by matentzn** [comment]
Re: orphacodes, best as @pnrobinson or look here: https://hpo.jax.org/app/download/annotation

I don't really know the answer to that question.

**Comment by julienN75** [comment]
Thank you the query works fine.

However, on HPO website we have access to orpha & omim codes (for instance under "Disease Associations" here : https://hpo.jax.org/app/browse/term/HP:0003493). That is why I don't understand why you can't get them with the query.

**Comment by pnrobinson** [comment]
I think it might be difficult to do what @julienN75 is asking. We can provide a list of all of the antibody terms with HP ids if that would be helpful. Note that we cannot add the new annotations to Orphanet diseases, of course, the Orphanet team needs to do that. 
Therefore, what we can provide is this


    hpo code
    label
    synonym
    description
    pmid

but not "orpha code"

**Comment by julienN75** [comment]
Thank you both for your answers. I will try to find another way to get the orpha codes.

I noticed some weird information in the pmid column from the query you provided @matentzn. I have rows containing "HPO:probinson" or "ORCID:0000-0002-3387-1836" which are not pmids. Do you know why ?

**Comment by matentzn** [comment]
I filtered them for you here:
https://api.triplydb.com/s/lcflTRR5a

**Comment by julienN75** [comment]
This data will be used for a project to be followed by a scientific paper. After discussing with @MaudDLVT, I actually need those rows even though they don't have a valid pmid.

Furthermore we don't understand why we only get 154 entries whereas you provided 244 entries for #7198. Basically what I would like is the 244 entries with the following columns :
- hpo code
- label
- synonym
- definition
- pmid

Thank you in advance !

**Comment by matentzn** [comment]
This should cover you then:

https://github.com/obophenotype/human-phenotype-ontology/blob/master/src/ontology/reports/autoimmune-antibody-report.tsv

If you are happy with this table, please also close the ticket!

**Comment by julienN75** [comment]
That is exactly what I needed. Thank you very much for your help !

**Comment by matentzn** [comment]
No problem :)
